models/actions/run_job.go

```diff
@@ -131,6 +131,11 @@ func UpdateRunJob(ctx context.Context, job *ActionRunJob, cond builder.Cond, col
 	}
 	if runStatus.IsDone() {
 		run.Stopped = timeutil.TimeStampNow()
+	} else if runStatus.IsWaiting() {
```

Comment:
Incorrect, `runStatus` is the status of `Run`, not `Job`.

I think it should be

```go
	if affected != 0 && util.SliceContains(cols, "status") && job.Status.IsWaiting() {
		// increaseTaskVersion
	}
```